wood.dan.design.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "wood.dan.design",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "wood.dan.design",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+      "name": "blueprint",
```

Comment:
this needs a cleaniup rule to remove it completely as it is a build tool.

Reply:
Do you know of any examples for this?

Reply:
just add the path to the cleanup section?